libnmstate/nm/applier.py

```diff
@@ -146,7 +148,7 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
 def _get_new_ifaces(con_profiles):
     ifaces_without_device = set()
     for con_profile in con_profiles:
-        ifname = con_profile.get_interface_name()
+        ifname = con_profile.profile.get_interface_name()
```

Comment:
The `interface_name` should be a property of the ConnectionProfile object.

Reply:
Done